eng/Versions.props

```diff
@@ -7,12 +7,17 @@
     <MinorVersion>0</MinorVersion>
     <PatchVersion>0</PatchVersion>
     <SdkBandVersion>8.0.100</SdkBandVersion>
-    <PackageVersionNet7>7.0.5</PackageVersionNet7>
+    <!-- note: when bumping previous package versions on a release branch, make sure to
+               set $(WorkloadsTestPreviousVersions)=false, so those manifests won't be installed
+               during testing. -->
+    <PackageVersionNet7>7.0.6</PackageVersionNet7>
     <PackageVersionNet6>6.0.$([MSBuild]::Add($([System.Version]::Parse('$(PackageVersionNet7)').Build),11))</PackageVersionNet6>
     <PreReleaseVersionLabel>preview</PreReleaseVersionLabel>
     <PreReleaseVersionIteration>5</PreReleaseVersionIteration>
     <WorkloadVersionSuffix Condition="'$(PreReleaseVersionLabel)' != 'release'">-$(PreReleaseVersionLabel).$(PreReleaseVersionIteration)</WorkloadVersionSuffix>
     <SdkBandVersionForWorkload_FromRuntimeVersions>$(SdkBandVersion)$(WorkloadVersionSuffix)</SdkBandVersionForWorkload_FromRuntimeVersions>
+    <!-- set to false for release branches -->
+    <WorkloadsTestPreviousVersions Condition="'$(WorkloadsTestPreviousVersions)' == ''">false</WorkloadsTestPreviousVersions>
```

Comment:
Why not set this based on `'$(DotNetFinalVersionKind)' == 'servicing'` which is defined below?

Reply:
@lewing is there a servicing scenario where we *can* run WBT?

Reply:
in theory we can run them if PackageVersionNet7 <= netcoreapp.LatestRuntimeFrameworkVersion

Reply:
@ViktorHofer not sure we can do anything right now.  Best to leave it as is and think more about it.